apps/payments/next/app/[offeringId]/error/page.tsx

```diff
@@ -70,96 +31,91 @@ export default async function Error({ params }: { params: CheckoutParams }) {
+            {l10n.getMessage('sub-guarantee')?.value?.toString() ||
+              '30-day money-back guarantee'}
+          </span>
         </div>
       </header>
 
-      {/* body */}
-      <article className="component-card border-t-0 mb-6 pt-4 rounded-t-lg text-grey-600 px-4 pb-12">
-        {/* main content */}
-        <section className="flex flex-col items-center justify-center">
-          {/* error icon hidden to screen readers as this information does not add to the page */}
-          <div className="mt-20 mb-12 mx-auto">
-            <ErrorIcon />
-          </div>
+      <section
+        className="payment-panel hidden tablet:block"
+        aria-label="Purchase details"
+      >
+        <PurchaseDetails
+          interval={cart.interval}
+          invoice={cart.nextInvoice}
+          purchaseDetails={contentful.purchaseDetails}
+        />
+      </section>
 
-          {/* error copy retrieved from card data */}
-          <div className="py-0 text-grey-400 mb-18 px-20 text-center">
-            <p className="mb-4 text-sm">
-              Your transaction could not be processed. Please verify your credit
-              card information and try again.
-              {/*
-                  Note: commented out as this portion is not part of main tasks
-                  {cart.errorReasonId}
-                  {l10n.getMessage(cart.errorReasonId)?.value?.toString() ||
-                'Your transaction could not be processed. Please verify your credit card information and try again.'}
-              */}
-            </p>
-          </div>
+      <div className="page-body rounded-t-none tablet:rounded-t-long">
+        <section
+          className="page-message-container h-[640px]"
+          aria-label="Payment error"
+        >
+          <Image src={errorIcon} alt="" className="mt-16 mb-10" />
+          <p className="page-message px-7 py-0 mb-4 ">
+            {l10n
+              .getMessage(getErrorReason(cart.errorReasonId).messageFtl)
+              ?.value?.toString() || getErrorReason(cart.errorReasonId).message}
+          </p>
 
-          {/* Try again button */}
           <Link
-            className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-3 px-4 rounded w-full text-center"
-            href="/123done/checkout?interval=monthly"
+            className="page-button"
+            href={`/${params.offeringId}/checkout?interval=monthly`}
           >
-            Try again
+            {l10n
+              .getMessage(getErrorReason(cart.errorReasonId).buttonFtl)
+              ?.value?.toString() ||
+              getErrorReason(cart.errorReasonId).buttonLabel}
           </Link>
-
-          {/*
-              Note: accessibility issue in T&P unrelated to error page - fails best practice but minor
-              Error: header levels should increase by one (missing h3)
-              Additionally, this would probably be a footer to the article
-          */}
-          <TermsAndPrivacy
```

Comment:
Aligned with revisiting the a11y issues with the T&P component.

Reply:
It's out of scope for this ticket but would definitely like to revisit.